platform/servicepulse.include.md

```diff
@@ -1,3 +1,5 @@
 ServicePulse is a web application designed for administrators. It provides a clear, near real-time, high-level overview of how a system is currently functioning.
 
-Notifications will occur when an endpoint is offline or when a message fails. [Custom checks](/monitoring/custom-checks/) can be built to get alerts for specific scenarios. The interface supports performing common operations for failure recovery, such as retrying failed messages. In order to display and handle them in a custom way, see [Using ServiceControl Events](/servicecontrol/contracts.md).
+Notifications are shown when an endpoint is offline or when a message fails. [Custom checks](/monitoring/custom-checks/) can be used to provide notifications for more specific scenarios. The ServicePulse UI provides common operations for failure recovery, such as retrying failed messages.
+
+NOTE: Further custom behaviours for message failures and other events can be acheived using [ServiceControl events](/servicecontrol/contracts.md).
```

Comment:
These two paragraphs make it sound like I can get a text message or email when message fails and that some hand-wavy custom behaviors can be created to do mysterious things. I think after reading this it would be a big disappointment to find out that ServicePulse is read-on-demand only, and that to get any real _notifications_ you have to use the ServiceControl events.

Reply:
I'll re-work something, which also takes into account the more advanced monitoring. I think we can leave out the stuff about ServiceControl events. In advanced scenario that can be discovered when someone digs deeper into the platform. This page is intended to be high level.

Reply:
@DavidBoike please take a look at the latest commit to see if it resolves this.

Reply:
LGTM